spec/requests/user_spec.rb

```diff
@@ -7,4 +7,11 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe 'GET /user/:id' do
+    it 'returns http success' do
+      get '/users/1'
+      expect(response).to have_http_status(:success)
+    end
+  end
```

Comment:
- [ ] Kindly add more tests in to check for templates and text from the pages.

Reply:
- [ ] Those tests have already been created in a separate folder `view`